docker/entrypoint.sh

```diff
@@ -75,6 +72,11 @@ EOF
     fi
 fi
 
+if [ "${KRILL_USE_TA}" == "true" ]; then
```

Comment:
@timbru: We should decide what we want to do here. Released versions of Krill until now supported a KRILL_USE_TA environment variable and when the Docker image detected that the env var was set it used it to set the equivalent but no longer support `use_ta = true` setting in krill.conf. In this edit I kept support for KRILL_USE_TA but made it set the new env vars that need to be set to activate TA + testbed mode, the nearest equivalent to what we had before. Keeping support for KRILL_USE_TA in the Docker image but not in Krill itself feels confusing and wrong. We could just remove this if block and advise people that if they want to use the TA they have to pass `-e KRILL_TESTBED_RSYNC=...` and `-e KRILL_TESTBED_RRDP=...` when invoking `docker run` ? What do you think?

Reply:
I think:

Remove this if block and advise people that if they want to use the TA they have to pass -e KRILL_TESTBED_RSYNC=... and -e KRILL_TESTBED_RRDP=... when invoking docker run.

Alternatively we could have a testbed binary which takes uris as arguments in addition to krill, or we could have an API / CLI endpoint to set up the testbed on a running Krill instead (and fail in case it's not a clean install). But to be honest, given the amount of use of TESTBED I can live with the env approach for now.

Reply:
I'll remove the block.